src/pfe/iterative-dev/idc-java/IDC/src/org/eclipse/codewind/iterdev/IDCContext.java

```diff
@@ -164,7 +164,10 @@ public String getAppId() {
 	}
 	
 	public String getLocalWorkspaceOrigin() {
-		if(appDb.get(Constants.DB_WORKSPACE_ORIGIN) != null) {
+		if (System.getenv("NODE_ENV").equals("test")) {
```

Comment:
Do you want to put this change in now or with the Liberty tests? asking because dont want src code to break since we havent started testing liberty projects

Reply:
I think spring project also uses the same code path? Either ways I think we can use it because I will be working on Liberty next and it will be required for that

Reply:
Spring does not use idc.java code path.

But since this is an if statement for test and you're going to use Liberty next, ok.